services/forms/repo_form.go

```diff
@@ -604,7 +604,7 @@ type MergePullRequestForm struct {
 	MergeMessageField      string
 	MergeCommitID          string // only used for manually-merged
 	HeadCommitID           string `json:"head_commit_id,omitempty"`
-	ForceMerge             *bool  `json:"force_merge,omitempty"`
+	ForceMerge             bool   `json:"force_merge,omitempty"`
```

Comment:
But then it's a break change?

Reply:
I don't think so, the default value `false` has the same effect as `nil` when there's no `force_merge` in API request or form request.